site/cypress/e2e/Cypress-Gradeable/late_submission_warning_messages.spec.js

```diff
@@ -118,6 +118,38 @@ const SubmitAndCheckMessage = (gradeable_type, upload_file1, invalid_late_day, v
     cy.logout();
 };
 
+
+it('Check for a clock message near daylight savings times', () => {
+    /*
+    First we set the date to a date close to the spring daylight savings time and then check that the message appears.
+    Then we change the date to a month later, and check that the "daylight" message doesn't appear.
+    */
+
+    const dateToSet = new Date();
+    const daylight_spring = new Date('2024-03-10');
```

Comment:
daylight spring and fall should not be hard coded to some date. Could you find a way to update using some dependancies?